galpy/potential/__init__.py

```diff
@@ -10,6 +10,7 @@
     DehnenSmoothWrapperPotential,
     DiskSCFPotential,
     DoubleExponentialDiskPotential,
+    EarthPREMPotential,
```

Comment:
```suggestion
    EarthPREMPotential,
    SymbolicSphericalPotential,
```
Also expose the base symbolic potential so people can implement their own one easily.